manifest-schema.graphql

```diff
@@ -6,6 +6,10 @@ type Schema {
   file: File!
 }
 
+type Manifest {
+  file: File!
+}
+
```

Comment:
I'd call this `ExternalMutations` and the inline `MutationsManifest` -> `InlineMutations`.